### PR TITLE
Added Location Selector feature to the map + tests

### PR DESCRIPTION
**Pull request by imaneb942**
- Worked on the first screen of the creation of an itinerary
- Scaffold with the bottom bar set to be the location picker, the user will be able to pick two points to create an itinerary
- the feature of those two points being displayed on the map are not a task of mine
- @ethangraham2001 worked on the map UI, waypoints and popUp screen
- the UI task that I worked on currently looks like this (: see image)
- My next goal is to have the freedom to add more that two points and be able to swap the points of wanted
-
image

### CODE REVIEW

File: app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    createItineraryViewModel =
+        CreateItineraryViewModel(itineraryRepository, directionsRepository, userLocationClient)
+    createItineraryViewModel.startNewItinerary(dummyProfile.userUid)
+
+    profileViewModel = ProfileViewModel(profileRepository, imageRepository)
+  }
+
+  @Test
+  fun dummy() {
```
Comment (ethangraham2001) [inline app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt:105]: Can we remove the dummy test?

File: app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt
Comment (ethangraham2001) [inline app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt:1]: I think the filename should be renamed. Perhaps LocationSelectorTest.kt? Alternatively, the tests could be placed in CreateItineraryMapTest.kt

**Review by ethangraham2001 — CHANGES_REQUESTED** [review]
Just need to remove the dummy test and perhaps relocate the tests / rename the file to avoid confusion

File: app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt (hunk 1 above)
Comment (imaneb942) [reply app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt:105]: yes my bad, forgot to take it off, it is removed now

File: app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt
Comment (imaneb942) [reply app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt:1]: Thanks, the file has been renamed

Comment (engjellismaili) [inline app/src/test/java/com/github/wanderwise_inc/app/ui/creation/createItineraryMapUITest.kt:1]: Lgtm, just remember that you can use directly "when" and "mock" by importing them

**Review by engjellismaili — COMMENTED** [review]
Lgtm well done

**Comment by engjellismaili** [comment]
Don't forget to run Ktfmt before pushing.

**Review by ethangraham2001 — APPROVED** [review]
Thanks for addressing the changes I suggested. Lgtm

**Pull request merged**